tests/app/App.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { describe } from "vitest";
+import "@testing-library/jest-dom";
+import { render } from "../test-utlis";
+import App from "../../src/App";
+import { fireEvent, screen } from "@testing-library/react";
+
+describe("react router functionality", async () => {
+  it("should navigate between two routes and render corresponding components", () => {
+    render(<App />);
+
+    expect(screen.getByText("hej, tu PlantMomma")).toBeInTheDocument();
```

Comment:
nothing's wrong here, bu it's usually more readable when you put the expression in between `expect` in a separate const.

Reply:
also, whenever you can use it - `.toBeVisible()` gives more certainty than `.toBeInTheDocument()`